templates/tdarr.xml

```diff
@@ -3,17 +3,19 @@
   <Name>tdarr</Name>
   <Repository>haveagitgat/tdarr</Repository>
   <Registry>https://hub.docker.com/r/haveagitgat/tdarr/</Registry>
-  <Network>bridge</Network>
+  <Network>host</Network>
```

Comment:
Any real reason its in host? just catched that

Reply:
Just makes it a bit easier to setup as with network bridge you have to expose the ports explicitly.

Reply:
But you already have set up those port in the template? I would like to avoid host networking as long as possible.

Reply:
Ok switched it to bridge.